data_dictionary.md

```diff
@@ -0,0 +1,37 @@
+
+# Example Activity Stream Log
+
+```{"session_duration":1635,"locale":"en-US","ip":"10.192.171.13","date":"2016-03-07",
```

Comment:
totall a nit but this would be a bit more readable as

```json
{
  "session_duration": 1635
   ...
}
```

i.e. with `json` after the three back ticks